.github/workflows/release.yml

```diff
@@ -12,19 +12,19 @@ jobs:
     steps:
       - name: Check out code
         uses: actions/checkout@v5
-      - name: Install nodejs
-        uses: actions/setup-node@v5
-        with:
-          node-version-file: .tool-versions
-          registry-url: "https://registry.npmjs.org"
 
       - name: Install pnpm
         uses: pnpm/action-setup@v4
+
+      - name: Install nodejs
+        uses: actions/setup-node@v6
         with:
-          version: latest
+          cache: pnpm
+          node-version-file: .tool-versions
+          registry-url: "https://registry.npmjs.org"
 
       - name: Install dependencies
-        run: pnpm install
+        run: pnpm install --frozen-lockfile
```

Comment:
Ensures lockfile is in sync with package.json; otherwise fails CI (which is good because the result would be unpredicatable)

Reply:
Turns out that `--frozen-lockfile` is enabled by default in CI. Docs: https://pnpm.io/cli/install#--frozen-lockfile

The arg is not needed.